helpers/argumentParsers.go

```diff
@@ -1833,6 +1833,673 @@ func ParseBPFProgType(rawValue uint64) (BPFProgType, error) {
 	return v, nil
 }
 
+// BPFHelpers is an enum as defined in https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h
```

Comment:
This statement is a bit ambiguous.

Perhaps,

`// BPFHelpers is an enumeration of BPF helpers defined in https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h`

WDYT?

Reply:
sure!

i copied it from the `BPFProgType`: https://github.com/aquasecurity/libbpfgo/blob/main/helpers/argumentParsers.go#L1709

i'll change both of them